lxd-p2c/main.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/lxc/lxd/shared/version"
+)
+
+type cmdGlobal struct {
+	flagVersion bool
+	flagHelp    bool
+}
+
+func main() {
+	app := &cobra.Command{}
+	app.Use = "lxd-p2c"
+	app.Short = "Physical to container migration tool"
+	app.Long = `Description:
+  Physical to container migration tool
+
+  This tool lets you turn any Linux filesystem (including your current one)
+  into a LXD container on a remote LXD host.
```

Comment:
Only remotely?

Reply:
Sounds like locally could be neat as well.

Reply:
Well, nothing preventing you from running this against localhost, but as most people will use / as the source, this would be a pretty good way to shoot yourself in the foot (recursive download loop effectively).